main-command/src/main/scala/sbt/BasicKeys.scala

```diff
@@ -39,6 +39,12 @@ object BasicKeys {
                                  "The wire protocol for the server command.",
                                  10000)
 
+  val suppressServer =
```

Comment:
To avoid double negatives, I would say we should keep the same name `autoStartServer`.

Reply:
Sounds good! I've reworked it from `false`-defaulting `suppressServer` to `true`-defaulting `autoStartServer`.